locallibrary/catalog/views.py

```diff
@@ -1,8 +1,16 @@
+import datetime
 from django.shortcuts import render
 from django.views import generic
 # Create your views here.
 from .models import Book, Author, BookInstance, Genre
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.decorators import login_required, permission_required
+from django.shortcuts import get_object_or_404
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+from catalog.forms import RenewBookForm
+from django.views.generic.edit import CreateView, UpdateView, DeleteView
+from django.urls import reverse_lazy
```

Comment:
cùng packed thì gom lại e nhé
```suggestion
from catalog.forms import RenewBookForm
from django.views.generic.edit import CreateView, UpdateView, DeleteView
from django.urls import reverse_lazy, reverse
```

Reply:
Dạ e sửa r ạ